src/Language/Mimsa/Project/Helpers.hs

```diff
@@ -0,0 +1,44 @@
+{-# LANGUAGE TupleSections #-}
+
+module Language.Mimsa.Project.Helpers
+  ( fromItem,
+    fromType,
+  )
+where
+
+import qualified Data.Map as M
+import qualified Data.Set as S
+import Language.Mimsa.Store.ExtractTypes
+import Language.Mimsa.Types.Identifiers
+import Language.Mimsa.Types.Project
+import Language.Mimsa.Types.Store
+
+----------
+
+fromItem :: Name -> StoreExpression ann -> ExprHash -> Project ann
```

Comment:
Taken from `Mimsa/Actions` which is getting too big, these are essentially fancy smart constructors so should live with other Project stuff.